apps/events-helsinki/src/pages/articles/index.tsx

```diff
@@ -113,7 +113,7 @@ export default function ArticleArchive({
         navigation={<Navigation page={page} />}
         content={
           <SearchPageContent
-            title={t('cms:archiveSearch.title')}
+            title={t<string>('cms:archiveSearch.title')}
```

Comment:
In my tests this was not needed

```suggestion
            title={t('cms:archiveSearch.title')}
```